blocks/right-rail-block/index.story.jsx

```diff
@@ -30,18 +36,57 @@ const layoutItemStyles = {
 		textAlign: "center",
 	},
 };
-const getStyles = (name) => {
-	const defaults = layoutItemStyles.default;
 
-	return { ...defaults, ...layoutItemStyles[name] };
+const layoutItemStylesCommerce = {
```

Comment:
We can use the same styles defined - we shouldn't need to add duplicated styles

Reply:
It was used before, and It allows to have a bit more fidelity to convey the commerce use case.